### DISCUSSION TITLE
[v3] noscript Tag triggert on wire:navigate / new navigation

### DISCUSSION THREAD

**Discussion by schonhoff** [livewire/livewire#6279] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.7 | v3.0.1

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

Hey all,

I tried out the new Version after some bugs fixed after the beta release version. In my app I'm using a <noscript> html tag to inform the users that they need to activate their Javascript.
This is the code in my app.blade.php head:

```blade
<head>
    [...]
    <noscript> <meta http-equiv="refresh"  content="0;url={{ route('no-javascript') }}"> </noscript>
    [...]
</head>
```

This code get triggered in the new Livewire Version, but I don't know why. I see the Livewire Full-Page component for a blink of an eye, but after that it redirects to the noscript route mentioned above. I'm also using the redirect navigate functionality mentioned in the docs (https://livewire.laravel.com/docs/navigate#redirects). This is how my redirect looks:

```php
public function submit(): void
{
    $this->redirect(route('component.abc'), navigate: true);
}
```

If I'm remove the `navigate: true` it will work. If I'm removing the `<noscript>`-tag it works. But I want to inform my users that their browser has some configuration not compatible with my app and I would love to use the new navigate functionality. Maybe someone can take a look at it?

Thanks and very good job so far!

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by schonhoff** [comment]
I tried it with the current stable version. Sadly it isn't fixed. Maybe someone can tell me what I need to do or another implementation of the noscript tag.

**Comment by robdekort** [comment]
This indeed sounds like a bug, I'm running into the same issue. It seems adding `x-persist` on the noscript tag is a workaround.

**Reply by schonhoff** [reply]
I will try this. Thanks! Hopefully it will be fixed in Livewire :-)

**Reply by schonhoff** [reply]
After changing our entire app to Livewire v3 I can say that it won't work for me :(

**Comment by schonhoff** [comment]
This is still a bug. The workaround is not working for me. Maybe @nuernbergerA or @calebporzio can have a look into this. 
My guess is, that the noscript tag isn't ignored by fetching the new script tags maybe on this line:
https://github.com/livewire/livewire/blob/main/js/plugins/navigate/page.js#L12
or 
https://github.com/livewire/livewire/blob/main/js/plugins/navigate/page.js#L34

But I'm not confident enough with Livewire/Alpine to try out the differents :(
